test/parallel/xyk-pallet.L1asset.register.update.test.ts

```diff
@@ -209,8 +192,10 @@ describe("update L1AssetData-", () => {
       });
```

Comment:
plz validate that the the update did not happen by quering the storage and validating the fields.

Reply:
Done

Reply:
nice!